src/v0/destinations/facebook_conversions/utils.js

```diff
@@ -79,16 +79,31 @@ const validateProductSearchedData = (eventTypeCustomData) => {
   }
 };
 
+const getProducts = (message, category) => {
+  let products = message.properties?.products;
```

Comment:
here we are assuming message.properties can be null and later we use it [message.properties] so what happens if this is null?